parser/src/main/java/com/linkedin/avroutil1/parser/avsc/AvscParser.java

```diff
@@ -400,10 +400,11 @@ private AvroNamedSchema parseNamedSchema(
                             }
                             LiteralOrIssue defaultValueOrIssue = parseLiteral(fieldDefaultValueNode, defaultValueExpectedSchema, fieldName.getValue(), context);
                             if (defaultValueOrIssue.getIssue() == null) {
-                                //TODO - allow parsing default values that are branch != 0 (and add an issue)
                                 defaultValue = defaultValueOrIssue.getLiteral();
                             } else {
                                 context.addIssue(defaultValueOrIssue.getIssue());
+                                defaultValue = new AvscUnparsedLiteral(fieldDefaultValueNode,
+                                    defaultValueOrIssue.getIssue().getLocation());
```

Comment:
We don't really handle issues properly so better to set the `defaultValue` to an unparsed literal to make sure the schema throws when trying to write